docs/lib/datastore/real-time.md

```diff
@@ -0,0 +1,12 @@
+---
+title: Real time
+description: Learn more about how DataStore handles data changes in real-time.
+---
+
+## Observe Data in real-time
+
+You can subscribe to changes on your Models. This reacts dynamically to updates of data to the underlying Storage Engine, which could be the result of GraphQL Subscriptions as well as Queries or Mutations that run against the backing AppSync API if you are synchronizing with the cloud.
+
+<inline-fragment platform="js" src="~/lib/datastore/fragments/js/data-access/observe-snippet.md"></inline-fragment>
+<inline-fragment platform="ios" src="~/lib/datastore/fragments/ios/data-access/observe-snippet.md"></inline-fragment>
+<inline-fragment platform="android" src="~/lib/datastore/fragments/android/data-access/observe-snippet.md"></inline-fragment>
```

Comment:
I like the fact we have a section dedicated to real-time. My suggestion would be to move these fragments to their own folder (move from `data-access/` to `real-time/` to match the page it belongs to)

Reply:
Thanks, updates pushed based on this feedback.